src/os/bluestore/BlueFS.cc

```diff
@@ -586,6 +586,19 @@ int BlueFS::_open_super()
   if (r < 0)
     return r;
 
+  if (bdev[BDEV_DB]->is_shared() && bl.is_zero()) {
```

Comment:
More complex alternative could be to e.g. extend `bluefs_super_t` with flag saying that particular instance is shadow of true super block and thus should be never used. The shadow super block would be written (in multi-bdev deployments) to `BDEV_SLOW` then.